include/clang/AST/Expr.h

```diff
@@ -2798,7 +2798,10 @@ class BoundsExpr : public Expr {
 /// classes).
 class CastExpr : public Expr {
 private:
-  enum { OP, BOUNDS, END_EXPR = 2 };
+  // BOUNDS - declared bounds of the result of the cast expression
+  // CASTBOUNDS - inferred bounds of cast expression
```

Comment:
Could you change the comment for CASTBOUNDS to say 'normalized version of BOUNDS'?